client/js/widgets/widget.js

```diff
@@ -25,6 +25,7 @@ export class Widget extends StateManaged {
       css: '',
       movable: true,
       movableInEdit: true,
+      clickable: true,
```

Comment:
This default feels weird because I would assume only types that actually do stuff on click are clickable. I would set it to false in here and set it to true in each of the clickable types.

Reply:
I agree and made the changes.